geowebcache/core/pom.xml

```diff
@@ -149,6 +178,12 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
       <version>${httpcomponents.version}</version>
+      <exclusions>
```

Comment:
Same reaction as in GeoTools... is httpclient written against commons-logging? If it is, then we'd see failures at deploy, if the expected API is not in the classpath.

Reply:
I was working hard to get only one commons-logging in the mix, and using the one provided by `log4j-jcl`.